### PR TITLE
feat: supports to logging apisix_latency and upstream_latency.

### PR DESCRIPTION
**Pull request by jagerzhang**
### What this PR does / why we need it:
supports to logging apisix_latency and upstream_latency(copy from tapisix by [@kyroslin](https://github.com/kyroslin)): https://github.com/apache/apisix/issues/5910

Discuss: https://github.com/apache/apisix/discussions/6010

### Pre-submission checklist:
The original code is used directly without adding other new logic.

### CODE REVIEW

File: apisix/utils/log-util.lua [hunk 1]
```diff
@@ -96,6 +96,8 @@ local function get_full_log(ngx, conf)
         }
     end
 
+    local latency, upstream_latency, apisix_latency = _M.latency_details_in_ms(ctx)
```
Comment (spacewander) [inline apisix/utils/log-util.lua:99]: Better to refactor the method and avoid calling the method like `= latency_details_in_ms(ctx)`

**Review by spacewander — COMMENTED** [review]
Let's update test to make CI pass

File: apisix/utils/log-util.lua [hunk 2]
```diff
@@ -120,7 +122,9 @@ local function get_full_log(ngx, conf)
         consumer = consumer,
         client_ip = core.request.get_remote_client_ip(ngx.ctx.api_ctx),
         start_time = ngx.req.start_time() * 1000,
-        latency = (ngx_now() - ngx.req.start_time()) * 1000
+        latency = latency,
+        upstream_latency = upstream_latency,
+        apisix_latency = apisix_latency
```
Comment (zhendongcmss) [inline apisix/utils/log-util.lua:127]: I think use `apisix_latency` is confused, refer another discussment https://github.com/apache/apisix/issues/6029.
The `apisix_latenct` includes client sending data via network, not apisix exclusive. From my experience, it is large when tranfer big file.

Comment (tzssangglass) [reply apisix/utils/log-util.lua:127]: Maybe in this PR do like 
`apisix_latency = apisix_latency - upstream_latency` to get the real apisix_latency, cc @spacewander

Comment (spacewander) [reply apisix/utils/log-util.lua:127]: The latency from the client exists through the whole request process.
For file upload, it affects the apisix latency much.
But for file download, it affects the upstream latency much.
Both latencies contain the latency from the client.
So it's OK to keep the current behavior.

File: apisix/utils/log-util.lua (hunk 1 above)
Comment (jagerzhang) [reply apisix/utils/log-util.lua]: ok, already update.

**Review by spacewander — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**